src/libraries/Common/src/Extensions/Logging/NullScope.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
```

Comment:
Why are these changes necessary for the code in "Common/src"? They are included in other project with nullability enabled already. So why does this change need them?

Reply:
They are included here as links, and I didn't know that nullability is enabled for them in other project.
Since there're no nullable annotations in them, it should be ok to revert those changes (will do this in a couple of hours).

Reply:
There's critical issue with having these changes, but it does clutter the file unnecessarily. So unless they are required for some reason, let's just revert them.

Reply:
Done. Also removed it from `NullExternalScopeProvider`. From what I've understood, it's not needed anymore